afs_ioc_migration/rename.py

```diff
@@ -0,0 +1,90 @@
+import dataclasses
+import os.path
+import typing
+
+T = typing.TypeVar("T")
+ORG = "pcdshub"
+
+
+@dataclasses.dataclass(frozen=True)
+class RepoInfo:
+    """
+    Information about where an AFS repository will be transferred.
+    """
+
+    name: str
+    github_url: str
+    github_ssh: str
+    afs_source: str
+    area: str
+
+    @classmethod
+    def from_afs(cls: type[T], afs_source: str) -> T:
+        """
+        Create a RepoInfo instance given the pull path to the afs source.
+        """
+        name = rename(afs_source)
+        area = name.split("-")[1]
+        github_url = f"https://github.com/{ORG}/{name}.git"
```

Comment:
Given the future migration of pcdshub to slaclab, `ORG` should probably be a parameter we can set, rather than an immutable module-level global

Reply:
Yes, I can set this up as a configurable now that this is more mature. It was helpful/simpler to hard-code it at the start.